backend/ibutsu_server/constants.py

```diff
@@ -1,5 +1,7 @@
 ALLOWED_TRUE_BOOLEANS = ["y", "t", "1"]
 HEATMAP_MAX_BUILDS = 40  # max for number of builds that are possible to display in heatmap
+COUNT_TIMEOUT = 0.5  # timeout for counting the number of documents [s]
+MAX_DOCUMENTS = 100000  # max documents for pagination, when apply_max=True
```

Comment:
I played around with different values here, 1e5 seemed the best for going to an arbitrary page in the range, but I'm open to other values.

Reply:
Sounds fine to me.